src/conversations/Conversations.ts

```diff
@@ -129,10 +141,16 @@ export default class Conversations {
       )
 
       // Load all conversations started after the newest conversation found
-      const newConversations = await this.updateV2Conversations(
+      let newConversations = await this.updateV2Conversations(
         latestConversation?.createdAt
       )
 
+      if (this.client.isGroupChatEnabled) {
+        newConversations = newConversations.concat(
+          await this.updateGroupConversations(latestConversation?.createdAt)
```

Comment:
If the latest 1:1 conversation is after the latest group conversation, won't this lead to a time period where group chats haven't been checked?

It would be a rare issue, since you _should_ be checking both at the same time. But if someone is flipping that `enableGroupChat` flag back and forth you can get into a weird situation. Not a blocker, but maybe something we should flag as a TODO.

Reply:
That makes sense. But you could still call conversations.list(), then turn on group chat. 

Not keeping me up at night, but if enough people start using it someone will do that and come to us with a very mysterious looking bug.

Reply:
Fixed in https://github.com/xmtp/xmtp-js/pull/381/commits/828b3dca945d5a907c45c25c85e489c703f000bb.